crates/re_viewer/src/ui/view_spatial/scene/scene_part/images.rs

```diff
@@ -324,21 +317,32 @@ impl ImagesPart {
             },
         };
 
-        // By default, we assign a radius to each point so that each point becomes a ball
-        // that just barely touches its diagonally-neighboring pixel-point.
-        // This means the point radius increases with distance.
-        // It also increases with the field of view, and decreases with the resolution.
+        // We want point radius to be defined in a scale where the radius of a point
+        // is a factor (`backproject_radius_scale`) of the diameter of a pixel projected
+        // at that distance.
+        let fov_y = intrinsics.fov_y().unwrap_or(1.0);
+        let pixel_width_from_depth = (0.5 * fov_y).tan() / (0.5 * h as f32);
         let radius_scale = *properties.backproject_radius_scale.get();
-        let point_radius_from_depth = 0.5_f32.sqrt() * intrinsics.fov_y().unwrap_or(1.0) / h as f32;
-        let point_radius_from_normalized_depth = radius_scale * point_radius_from_depth * scale;
-
-        // TODO(emilk): we get gaps between diagonally adjacent points without this hack. WHY???
-        let point_radius_from_normalized_depth = 1.1 * point_radius_from_normalized_depth;
+        let point_radius_from_world_depth = radius_scale * pixel_width_from_depth;
+
+        let max_data_value = if let Some((_min, max)) = ctx.cache.tensor_stats(tensor).range {
+            max as f32
+        } else {
+            // This could only happen for Jpegs, and we should never get here.
+            // TODO(emilk): refactor the code so that we can always clauclate a range for the tensor
```

Comment:
```suggestion
            // TODO(emilk): refactor the code so that we can always calculate a range for the tensor
```

Or did you mean "klauculate", which is the thing German thieving calculators (known as "Klaucuclator") do?